wordle-web/components/ValidWords.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-dialog v-model="show">
+    <v-card class="ma-auto" min-width="300">
+      <v-sheet :color="validWords.length === 0 ? 'unknown' : 'secondary'">
+        <v-card-title class="text-center"
+          >Valid Words
+          <v-btn

```

Comment:
Good implementation on the button with an infinite scroll sheet, I used a select menu, but your implementation looks much prettier.

Reply:
Thank you! I would love to implement a search feature like I saw others do.